dkms_prebuild.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
maybe remove `/usr/bin/env` or add it to `autogen.sh` too?

Reply:
I added it to `autogen.sh` in the latest commit. This is important because with distros merging `/usr/bin` and `/bin`, it is on the table that `/bin` disappears and then `sh` is only at `/usr/bin/sh`. But `/usr/bin/env` is stable.